setup.py

```diff
@@ -143,6 +143,7 @@ def get_cmdclass():
         ],
         'asreview.datasets': [
             'builtin = asreview.datasets:BuiltinDataGroup',
+            'benchmark = asreview.datasets:BenchmarkDataGroup',
```

Comment:
Will there be a difference between the built-in datasets and the benchmarkDataGroup? I can image we keep some datasets available in the front-end for workshops, but I can also image we simplify the code and keep only the latter group.

Reply:
Good one. I'm deprecating the built-in group. I was thinking about having a featured subset in LAB with the datasets we currently use a lot. In this way, we resolve both.

Reply:
O yes, that is a perfect idea!